### PR TITLE
docs(menu): add description

### PR DESCRIPTION
**Pull request by philibea**
## Type

- Documentation

### Summarise concisely:

Add documentation on each props and story.

### CODE REVIEW

File: src/components/Menu/__stories__/index.stories.mdx [hunk 1]
```diff
@@ -91,6 +116,8 @@ import { Box, Button, Icon, Modal, Touchable } from '../..'
 
 ## Modal
 
+<Description>This will show how to use a modal on Menu.Item.</Description>
```
Comment (vincentaudebert) [inline src/components/Menu/__stories__/index.stories.mdx:119]: This shows

File: src/components/Menu/__stories__/index.stories.mdx [hunk 2]
```diff
@@ -118,6 +145,8 @@ import { Box, Button, Icon, Modal, Touchable } from '../..'
 
 ## Advanced ( Work In Progress )
 
+<Description>Still Work In Progress. Add Sub Menu Item.</Description>
```
Comment (vincentaudebert) [inline src/components/Menu/__stories__/index.stories.mdx:148]: Add a sub menu item

File: src/components/Menu/__stories__/index.stories.mdx [hunk 3]
```diff
@@ -51,6 +72,10 @@ import { Box, Button, Icon, Modal, Touchable } from '../..'
 
 ## Borderless
 
+<Description>
+  Properties borderless avoid a bottom border on Menu.Item
```
Comment (vincentaudebert) [inline src/components/Menu/__stories__/index.stories.mdx:76]: Property `borderless` avoids

File: src/components/Menu/__stories__/index.stories.mdx [hunk 4]
```diff
@@ -25,6 +40,12 @@ import { Box, Button, Icon, Modal, Touchable } from '../..'
 
 ## Variants
 
+<Description>
+  A set of variant you can add on Menu.Item (danger, nav). You can use either
+  props onMenuItem : - `onClick` to define menu actions. - `to`to define as a
```
Comment (vincentaudebert) [inline src/components/Menu/__stories__/index.stories.mdx:45]: `to` to (small space missing)

Comment (vincentaudebert) [inline src/components/Menu/__stories__/index.stories.mdx:45]: on MenuItem

File: src/components/Menu/__stories__/index.stories.mdx [hunk 5]
```diff
@@ -1,11 +1,26 @@
...
 
 <Meta title="Components/Menu" component={Menu} />
 
 # Menu
 
+<Description>
+  A menu is a widget that offers a list of choices to the user, such as a set of
+  actions or functions. A menu is usually opened, or made visible, by activating
+  a menu button, choosing an item in a menu that opens a sub menu, or by
+  invoking a command, such as Shift + F10 in Windows, that opens a context
```
Comment (vincentaudebert) [inline src/components/Menu/__stories__/index.stories.mdx:19]: on Windows

**Review by vincentaudebert — COMMENTED** [review]
small fixes otherwise all good

**Review by matthprost — APPROVED** [review]

**Review by vincentaudebert — APPROVED** [review]

**Pull request merged**